client/src/components/UploadPreview.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const Display = styled.div`
+  height: 10rem;
+  width: 10rem;
+  border: solid 1px black;
+  margin-left: auto;
+  margin-right: auto;
+`;
+const IMG = styled.img`
```

Comment:
The naming is awkward because its just the html element in uppercase.
```suggestion
const PreviewImage = styled.img`
```